keras_retinanet/utils/config.py

```diff
@@ -35,6 +35,9 @@ def read_config_file(config_path):
     assert config_keys <= default_keys, \
         "Malformed config file. These keys are not valid: {}".format(config_keys - default_keys)
 
+    if 'pyramid_levels' in config:
+        assert('P' in config['pyramid_levels']), "pyramid levels specified by P key"
```

Comment:
Does this work? Afaik there is no `'P'` key in the pyramid levels config right? Did you intend to search for a key that starts with `'P'`?

Reply:
Well previously there was no options at all for pyramid levels config. The way I implemented formatting the config files is as follows:

```
[anchor_parameters]
...
[pyramid_levels]
P = 2 3 4 5 6 7
```

You are right that this may be opaque (would need to modify the example config.ini template found elsewhere on the repo) , so open to suggestion here if you have a preferred way.

Reply:
How about something a bit more descriptive, like `'levels'`?

Reply:
alright, done

Reply:
hello, just wanted to ping to see if there's any other requested changes.